app.js

```diff
@@ -1 +1,323 @@
 // Cookies needed per location projections
+
+var hours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm','6pm', '7pm', '8pm'];
```

Comment:
I replicated what was included in the assignment overview.

<img width="775" alt="screenshot 2016-06-21 10 56 50" src="https://cloud.githubusercontent.com/assets/5835129/16240667/1e177bae-379f-11e6-8f61-174c7cc99e4d.png">

Should I not include 8pm given the text description or mirror what was displayed in the example (e.g. shows 8pm)?